lib/engine.js

```diff
@@ -124,14 +131,15 @@ Engine.prototype._configureApp = function() {
   log.info('configuring service endpoints');
 
   let self = this;
+  const app = express();
   const routers = Server.Routes({
     config: this._config,
     storage: this.storage,
     network: this.network,
     mailer: this.mailer,
-    contracts: this.contracts
+    contracts: this.contracts,
+    client: this.client
```

Comment:
More specifically this is a redis client, there are a few clients already here. Could be more clear to use `redis` instead of `client`.